test/cases/ostree-ng.sh

```diff
@@ -559,6 +559,15 @@ sudo rm -f "$LIBVIRT_BIOS_IMAGE_PATH"
 ## Install, upgrade and test Edge image on UEFI VM
 ##
 ##################################################
+
+# Since virt-install 4.0.0, loader attribute can't be configured here,
+# otherwise it'll report "loader attribute 'readonly' cannot be specified
+# when firmware autoselection is enabled"
+if nvrGreaterOrEqual "virt-install" "4.0.0"; then
+    BOOT_UEFI_ARGS="uefi"
```

Comment:
@henrywang This might blow up on CS9 because of secure boot. If it happens, this is how to disable since virt-install 4:
https://github.com/osbuild/osbuild-composer/blob/main/test/cases/ostree-raw-image.sh#L521

Reply:
@ondrejbudai Thanks. Do you know the reason to disable secure boot? Thanks!

Reply:
But if I read the bug correctly this should only be needed on CS9, not on RHEL, right, but it seems that the code is unconditionally disabling secure boot based on the `virt-install` version. Wouldn't that mean we don't test secure boot at all anymore on newer versions (even on RHEL)?

Reply:
How about: Let's update the CS9 repos (I can include it in this PR even though it's not related) and change the secureboot condition to disable only for CS9 (regardless of `virt-install` version)?

Reply:
Opened issue https://github.com/osbuild/osbuild-composer/issues/3029 for this so we don't delay this PR and we can do it in a followup.